fMRIVolume/scripts/DistortionCorrectionAndEPIToT1wReg_FLIRTBBRAndFreeSurferBBRbased.sh

```diff
@@ -293,52 +305,53 @@ case $DistortionCorrection in
         fi
```

Comment:
Please convert preceding `log_Msg; exit 1` to an `log_Err_Abort`

Reply:
@mharms There are a number of these throughout the script. Do I convert all of them?

Reply:
Sure.  Might as well.

Reply:
I did.